functorch/writing_batching_rules.md

```diff
@@ -79,8 +79,6 @@ m.impl("trace", trace_decomp);
 ```
 In general, this reduces the performance, since instead of launching one kernel we're launching multiple. So, we generally try to avoid this option :)
 
-PS: There's a special class of operators in PyTorch called CompositeImplicitAutograd operators that can be decomposed without losing performance. We actually decompose those by default! (although we stop the decompositions that aren't tested in [BatchRulesStopDecomposition](functorch/csrc/BatchRulesStopDecomposition.cpp) for debuggability reasons).
```

Comment:
I couldn't find `BatchRulesStopDecomposition.cpp` so this paragraph is removed. Let me know if it is still needed.